pkg/driver/node.go

```diff
@@ -464,6 +473,16 @@ func findNvmeVolume(findName string) (device string, err error) {
 	return resolved, nil
 }
 
+// getVolumesLimit returns the limit of volumes that the node supports
+func (d *nodeService) getVolumesLimit() int64 {
+	ebsNitroInstanceTypeRegex := "^[cmr]5.*|t3|z1d"
+	instanceType := d.metadata.GetInstanceType()
+	if ok, _ := regexp.MatchString(ebsNitroInstanceTypeRegex, instanceType); ok {
```

Comment:
Feels its less error prune if we just maintain a hard coded list of nitro instance instead of regex. And define a function that checks if it is nitro based:

```go
func isNitroInstance(instanceType string) boolean 
```

How do you think?

See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-types.html#ec2-nitro-instances for a full list of Nitro instances.

Reply:
I'm OK with that, but I'm not sure if I understand that page well enough to come up with a reliable list. If nothing, the regex above comes from the in-tree implementation, so it seems to be a reasonable good start.